lib/filter-overlapping-peaks.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * This function de-dups overlapping peaks in each dataset from ChIP, DNase and ATAC-seq assays.
+ * @param {Array} Datasets list of datasets to filter
+ * @returns {Array} Return an array. Array[0] si DatasetsFiltered: datasets after filtering. Array[1] is DatasetsFilteredCount: the number of datasets that are filtered out.
+ */
+export default function filterOverlappingPeaks(Datasets) {
```

Comment:
Is it odd that `Datasets` and the other local variables are uppercase? Does that have a meaning in this project?